lib/field/use-field-like.ts

```diff
@@ -146,6 +161,17 @@ export const useFieldLike = <
     [formContext, props]
   );
 
+  const [value, _setValue] = useState(initialState: () => {
+    const initVal = (props.initialValue ?? initialValue) as UseFieldLikeProps<T,F,TT>["initialValue"]
+
+    runFieldValidation(validationFnName: "onMountValidate", initVal)
```

Comment:
This isn't quite right, it should be:

```suggestion
    runFieldValidation("onMountValidate", initVal)
```